app/controllers/form_controller.rb

```diff
@@ -102,6 +102,12 @@ def responses_for_page(page)
         result[question.id] = question_params
       end
 
+      if %w[checkbox].include?(question.type) &&
```

Comment:
This requires a bit of context. We can't simply leave the answer as null, since there's a field (`pregblank`) used for pre-existing "none of the above" answers to this question.

However, it's also too late to change the 24/25 form to add an exclusive "None of the above" checkbox. Instead, we need to make it so not filling out any checkboxes sets the `pregblank` field, hence the `default_answer` stuff here.

I do wonder whether it would be better to get rid of the abstraction here and just make it something we do for this specific question, as in bulk uploads.

Reply:
We set a lot of the derived values here `module DerivedVariables::LettingsLogVariables` would it be possible/make sense to do that as part of `set_derived_fields!` as well?

Reply:
Thanks for the suggestion - done!